sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -909,6 +911,11 @@ SIREPO.app.directive('logoutMenu', function(authState, authService, requestSende
             $scope.showAdmJobs = function() {
                 return authState.roles.indexOf('adm') >= 0;
             };
+
+            $scope.showJobsList = function() {
+                $('#' + panelState.modalId('jobsListModal')).prependTo("body");
```

Comment:
If you move the `<div data-jobs-list-modal="" data-title="Jobs" data-id="sr-jobsListModal-editor"></div>` to the commonFooter directive then you shouldn't need this line.